packages/endpoints-organizations-registrar/src/entities/organizations/adapters/send-email-notifications.js

```diff
@@ -162,14 +163,31 @@ const initSendEmailNotifications = (initCtx) => {
     const inviterOrganization = invitation?.inviterId
       ? await context.repos.organizations.findById(invitation.inviterId)
       : null;
-    await initCtx.sendEmail(
-      emailToSignatoryForOrganizationApproval({
+    const emailTemplateSignatoryApproval = compile(
+      readFileSync(`${__dirname}/email-template-signatory-approval.hbs`, {
+        encoding: 'utf8',
+      })
+    );
+    await initCtx.sendEmail({
+      subject: `${isReminder ? 'Reminder: ' : ''}${
```

Comment:
1. why would it ever be an reminder?
2. The subject should be extracted into a template like the message body?

Reply:
> why would it ever be an reminder?

Basically if it isn't the initial email sent during creation, and it is called by the lambda, then that flag is true. This is how the copy has been. Should I change to static subject?

@sloops77